packages/color-kit-block/src/EmptyView.tsx

```diff
@@ -0,0 +1,27 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Text } from '@frontify/fondue';
+
+type EmptyViewProps = {
+    paletteName?: string;
+};
+
+export const EmptyView = ({ paletteName = 'Select color palettes for them to appear here' }: EmptyViewProps) => {
```

Comment:
wdyt? or is this the design?
```suggestion
export const EmptyView = ({ paletteName = 'Select color palettes in the settings for them to appear here' }: EmptyViewProps) => {
```

Reply:
This is the current copy, not my call to change it 😄

Reply:
👍 :)